.travis.yml

```diff
@@ -21,14 +21,15 @@ services: docker
 env:
   matrix:
   - INSTANCE=ubuntu-1404
-  - INSTANCE=ubuntu-1604
+  # - INSTANCE=ubuntu-1604
```

Comment:
[I couldn't get it to build](https://travis-ci.org/maraaaa/chef-rocketchat/jobs/236632286). (Could have sworn it was something other than just sudo not being installed...)

It was more point of record than a suggestion of not targeting 16.04.

Actually, the whole reason for these changes is we wanted to run it on CentOS but couldn't even get it to work on Ubuntu.  So I figured get it to build somewhere then work from there.